source/client/client_worker_impl.cc

```diff
@@ -106,4 +106,4 @@ StatisticPtrMap ClientWorkerImpl::statistics() const {
 }
 
 } // namespace Client
-} // namespace Nighthawk
+} // namespace Nighthawk
```

Comment:
Some changes that seem unnecessary here and below

Reply:
Fixed.